src/common/license/src/license.rs

```diff
@@ -0,0 +1,15 @@
+use serde::Deserialize;
+use serde::Serialize;
+
+pub const LICENSE_PUBLIC_KEY: &str = r#"-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEGsKCbhXU7j56VKZ7piDlLXGhud0a
+pWjW3wxSdeARerxs/BeoWK7FspDtfLaAT8iJe4YEmR0JpkRQ8foWs0ve3w==
+-----END PUBLIC KEY-----"#;
+
+#[derive(Serialize, Deserialize)]
+pub struct LicenseInfo {
+    #[serde(rename = "type")]
+    pub type_: Option<String>,
```

Comment:
this could be `#type`?

Reply:
or naming it as `kind` to avoid conflict with the `type` keyword?